game.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body onmousedown="showCoords(event)">
+    <canvas id="main" width="360" height="360"></canvas>
```

Comment:
can you set width and height in the style?

Reply:
done